src/app/pages/loading-waiting-states/contextual-spinners/contextual-spinners.component.scss

```diff
@@ -1,19 +1,101 @@
 @import '~@pxblue/colors/palette.scss';
 
-.extended-fab-button {
-    width: 180px;
-    height: 56px;
-    padding: 0 26px;
-    border-radius: 29px;
-    transition-property: unset;
-    transition-duration: 1s;
-
-    .mat-icon {
-        margin-right: 8px;
+section.mat-typography {
+    padding: 16px 24px;
+    .example-section {
+        margin-bottom: 16px;
+    }
+    .show-text {
+        .text-container {
+            opacity: 1;
+            visibility: visible;
+            -webkit-transition: opacity 200ms, visibility 200ms;
```

Comment:
I don't think we need to have any browser prefixes in here...I think Angular handles that for us

Reply:
Can we define a sass variable for some of these repeated transition values?